### DISCUSSION TITLE
Help with a handler for a custom Editor (Android). Width is zero: only fits one character per line

### DISCUSSION THREAD

**Discussion by aarcoraci** [dotnet/maui#15916] [Q&A]
I'm creating a special version of the Editor and I'm having some problems on Android when using the component. The Editor is allowing just one or two characters per line (depending on the length of the character) and I believe this has to do with the width.

Here is the code for the handler:
```c#
public sealed partial class CustomEditorHandler : ViewHandler<CustomEditor, EditText>
{
    protected override EditText CreatePlatformView()
    {
        return new EditText(Context);
    }
    protected override void ConnectHandler(EditText platformView)
    {
        base.ConnectHandler(platformView);
    }

    static void MapIsKeyboardEnabled(CustomEditorHandler handler, CustomEditor editor)
    {
    }
}
```

If I set the width to an arbitrary value when creating the PlaftormView it "works". I have tried 
 
But no luck.

Any ideas ?

Screenshot shows how it only fits one or two characters. If I set width manually it "works" but I would like it to use it's container width.

![Screenshot_1687982084](https://github.com/dotnet/maui/assets/8377269/e44dfadc-ac47-44a4-8974-5cc58c2da1d6)

**Comment by aarcoraci** [comment]
I fixed the problem. My custom control definition has inheriting from Editor, I changed this to View and it seems to be working now

**Reply by yashaswinidc** [reply]
Hi @aarcoraci, Can you provide the solution, how you resolved this?

**Reply by aarcoraci** [reply]
We ended dropping MAUI in favor of flutter. A very positive change I may add.
